README.md

```diff
@@ -29,46 +29,28 @@ or
 $ rake rails:template LOCATION=https://raw.githubusercontent.com/sul-dlss/spotlight/master/template.rb
 ```
 
-Or do it manually:
+*During this process you will be prompted to enter an initial administrator email and password (this is a super-admin that can administer any exhibit in the installation).*
 
-Add these lines to your application's Gemfile:
+Change directories to your new application
 
 ```
-gem 'blacklight'
-gem 'blacklight-spotlight'
+$ cd app-name
 ```
 
-And then execute:
+Start solr
 
 ```
-$ bundle install
+$ rake jetty:start
 ```
 
-Then install Blacklight:
+Start rails
 
 ```
-$ rails generate blacklight:install
+$ rails s
 ```
 
-Then install Spotlight:
+Go to http://localhost:3000 in your browser.
 
-```
-$ rails generate spotlight:install
-$ rake spotlight:install:migrations
-$ rake db:migrate
-```
-
-Create an initial administrator:
-
-```
-$ rake spotlight:initialize
-```
-
-If you installed jettywrapper with Blacklight (above), install Spotlight's demo solr configuration and catalog controller configuration:
-
-```
-$ rake jetty:configure_solr
-```
 
 ## Configuration
 
```

Comment:
I think we can drop `Blacklight configuration` and `Solr Indexing`, which should be covered in the wiki.